packages/lib/src/components/Card/components/ClickToPay/components/CtPSection/CtPSection.scss

```diff
@@ -27,16 +30,16 @@
 }
 
 .adyen-checkout-ctp__section-header-scheme {
-    width: 30px;
-    object-fit: cover;
-}
-
-.adyen-checkout-ctp__section-header-divider {
-    background-color: black;
-    width: 1px;
-    height: 15px;
-    border-radius: 1px;
+    object-fit: none;
     margin-right: 6px;
+
+    &-mc {
+        width: 27px;
```

Comment:
The logos are SVG right? Why don't we control their size in the SVG?

Reply:
We don't have such control as the SVG's are external resources hosted in the checkoutshopper. In this case here, we are controlling size of the <img> html element which wraps the SVG's